docs/embeddable-uis/element/operation-approval.mdx

```diff
@@ -0,0 +1,427 @@
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Operation Approval Element
+
+<span className="subtitle-custom">
+The Operation Approval Element is a frontend component that enables users to request approval for specific actions in your application.
+<br/>
+When a request is made, it will display in the "Approval Management" Element, where users with the appropriate permissions can view all pending requests and approve or deny them.
+<br/>
+Once the request is approved or denied, the user will receive approval for the requested action.
+Additionally, a webhook will be triggered to notify your application about the approval or denial of the request.
+</span>
+<br/>
+<br/>
+
+<span style={{margin:'0 30px'}}>
+    <img src="/img/operation-approval/approvalRequest.png" alt="ar-element" style={{width:'70%'}}/>
+</span>
+
+## Overview
+
+The Operation Approval Element enables users to submit operation approvals using a dedicated user interface embedded in your application.
+This allows you, as the administrator, to restrict end-users from taking specific actions on objects and resources, and require an approval process before these actions are executed.
+After a user submits a request, it is sent for approval by an admin, who can either approve or deny it.
+
+This element is especially useful for critical actions in your system that require approval by a higher authority before execution.
+
+Using the "Operation Approval" Element is simple – just embed it into your Frontend application.
+Once embedded, users can request approval for restricted resources by clicking the "Request Approval" button.
+Later, users with the relevant permissions can view a list of all pending operation approvals in the "Approval Management" Element, where they can approve or deny them.
+
+**Example:**
+
+Let's take the example of a bank where users can transfer funds to other users.
+We want to ensure that all transfers above a certain amount are approved by a manager before they are executed.
+In this case, we can use the "Operation Approval" Element to allow users to request approval for large transfers.
+
+Upon receiving the operational approval, the manager reviews it to decide whether to approve this transfer request or not.
+
+TODO DIAGRAM
+
+[//]: # (![access-request]&#40;/TODO/ADD/IMAGE&#41;)
+
+
+## Creating and Connecting the Element
+
+### 1. In the [Elements Screen](https://app.permit.io/elements), under "Operation Approval", click on "Create Element".
+
+![create-a-r-element](/img/operation-approval/elementScreen.png)
+
+### 2. Select Resource from the dropdown menu, which will be the resource that connects to this element.
+
+![create-a-r-element](/img/operation-approval/approval-r-beforeSelectResource.png)
+:::note
+When the "Operation Approval" Element is created, it will create by default two roles: "_reviewer_" with the permission to review, approve and deny requests, and "_requester_" with the "operate" permission.
```

Comment:
the reviewer role display as Italic type and it is supposed to be display with _ before and after the "reviewer"

Reply:
also about the requester